### DISCUSSION TITLE
Node net in/net out sensors

### DISCUSSION THREAD

**Discussion by ishioni** [dougiteixeira/proxmoxve#344] [Feature request] [answered]
Can we get network usage sensors for nodes?

**Comment by Lipown** [comment]
I guess it is there - IN and OUT but it shows really weird numbers. Really unrealistically high numbers..

**Reply by tismondo** [reply]
Perhaps I've configured something wrong. I don't see net in/out for nodes, only individual CTs/VMs. I found my way here looking for it.... I'm wondering if there's a separate permission needed to get access to net traffic?

**Comment by dougiteixeira** [comment, marked as answer]
There is no network traffic information available in the Proxmox API, so it is not feasible to include this.
